docs/guide/debugging-c-cpp-controllers.md

```diff
@@ -34,6 +34,8 @@ The following example assumes that there is a problem with the "soccer\_supervis
 
 ### Using the GNU Debugger with a Controller
 
+On Windows you will need to install MSYS2 and the `mingw-w64-x86_64-gdb` package as indicated in the [optional dependencies](https://github.com/omichel/webots/wiki/Windows-Optional-Dependencies) of the [Windows installation instructions](https://github.com/omichel/webots/wiki/Windows-installation).
```

Comment:
That's probably not the only option (and probably not the simplest, lighter for the user), I would rephrase it as 'On Windows, you can install GDB with MSYS2 for example...'.